scipy/fft/_pocketfft/helper.py

```diff
@@ -144,17 +144,18 @@ def _fix_shape_1d(x, n, axis):
     return _fix_shape(x, (n,), (axis,))
 
 
-def _normalization(norm, forward):
-    """Returns the pypocketfft normalization mode from the norm argument"""
+_NORM_MAP = { None: 0, 'backward': 0, 'ortho': 1, 'forward': 2}
```

Comment:
This is the only actual functionality change.

Reply:
```
./scipy/fft/_pocketfft/helper.py:147:14: E201 whitespace after '{'
1       E201 whitespace after '{'
```